index.html

```diff
@@ -0,0 +1,539 @@
+  </section>
+  <section class="grid-part2">
+    <div class="grid2-art">
+      <h5>Guides</h5>
+      <h1>New frontend challenges to master</h1>
+      <ul>
+        <li><a href="#">CSS Grid Layout</a></li>
+        <li><a href="#">Pattern library</a></li>
+        <li><a href="#">Progressive Web Apps</a></li>
+        <li><a href="#">Switching to HTTPS</a></li>
+        <li><a href="#">HTTP/2 guidelines</a></li>
+        <li><a href="#">React & Vue.js</a></li>
+        <li><a href="#">Redux for designers</a></li>
+        <li><a href="#">Vue components</a></li>
+      </ul>
+    </div>
+    <div  class="grid2-art">
+      <h5>Guides</h5>
+      <h1>New frontend challenges to master</h1>
+      <ul>
+        <li><a href="#">CSS Grid Layout</a></li>
+        <li><a href="#">Pattern library</a></li>
+        <li><a href="#">Progressive Web Apps</a></li>
+        <li><a href="#">Switching to HTTPS</a></li>
+        <li><a href="#">HTTP/2 guidelines</a></li>
+        <li><a href="#">React & Vue.js</a></li>
+        <li><a href="#">Redux for designers</a></li>
+        <li><a href="#">Vue components</a></li>
+      </ul>
+    </div>
+    <div  class="grid2-art">
+      <h5>Guides</h5>
+      <h1>New frontend challenges to master</h1>
+      <ul>
+        <li><a href="#">CSS Grid Layout</a></li>
+        <li><a href="#">Pattern library</a></li>
+        <li><a href="#">Progressive Web Apps</a></li>
+        <li><a href="#">Switching to HTTPS</a></li>
+        <li><a href="#">HTTP/2 guidelines</a></li>
+        <li><a href="#">React & Vue.js</a></li>
+        <li><a href="#">Redux for designers</a></li>
+        <li><a href="#">Vue components</a></li>
+      </ul>
+    </div>
+    <div  class="grid2-art">
+      <h5>Guides</h5>
+      <h1>New frontend challenges to master</h1>
+      <ul>
+        <li><a href="#">CSS Grid Layout</a></li>
+        <li><a href="#">Pattern library</a></li>
+        <li><a href="#">Progressive Web Apps</a></li>
+        <li><a href="#">Switching to HTTPS</a></li>
+        <li><a href="#">HTTP/2 guidelines</a></li>
+        <li><a href="#">React & Vue.js</a></li>
+        <li><a href="#">Redux for designers</a></li>
+        <li><a href="#">Vue components</a></li>
+      </ul>
+    </div>
+    <div class="smashing-background">
+      <div class="slashing-news">
+        <h1><a href="#"> Smashing Newsletter</a></h1>
+        <p>Imagine you wrote a beautiful song and nobody discovered it.
+          That would make any cat sad. Don't let that happen!</p>
+        <img src="images/cat-with-trampet.svg" alt="">
+        <div class="email">
+          <h5>Your (smashing) email</h5>
+          <h3><a href="#">Subscribe →</a></h3>
+        </div>
+        <h6><a href="#">Useful tips for web designers.</a> Sent 2× a month.
+          You can unsubscribe any time — <em>obviously.</em></h6>
+      </div>
+    </div>
+    <p class="see-all2">See all Articles →</p>
+  </section>
+  <footer>
```

Comment:
You don't have to delete the text of the footer but you do need to change the background color to reflect the visual importance of the section.

Reply:
Done!